### PR TITLE
- regionService

### PR DESCRIPTION
**Pull request by rjschwei**
+ Port to Python 3
  + Add subjectAltName to generated certificate

### CODE REVIEW

File: regionService/cloud-regionsrv.spec [hunk 1]
```diff
@@ -17,20 +17,19 @@
...
 URL:            https://github.com/SUSE/Enceladus
 Group:          Productivity/Networking/Web/Servers
 Source0:        %{name}-%{version}.tar.bz2
 Requires:       apache2
-Requires:       apache2-mod_wsgi     >= 4.2.8
+Requires:       apache2-mod_wsgi-python3     >= 4.2.8
 Requires:       openssl
-Requires:       python
-Requires:       python-Flask
-Requires:       python-netaddr
```
Comment (bear454) [inline regionService/cloud-regionsrv.spec:32]: Don't need this dependency anymore?

File: regionService/srv/www/regionService/regionInfo.py [hunk 2]
```diff
@@ -134,12 +134,12 @@ def create_smt_region_map(conf):
         region_name_to_smt_data_map[section] = smtInfo
         for ipRange in region_public_ip_ranges.split(','):
```
Comment (smarlowucf) [inline regionService/srv/www/regionService/regionInfo.py:135]: Todo: Unrelated but note to fix case.

```ip_range```, ```smt_info``` etc

File: regionService/usr/sbin/genRegionServerCert [hunk 3]
```diff
@@ -51,22 +60,28 @@ def create_self_signed_cert(cert_dir, cert_file, cert_key, options):
     cert.get_subject().O = options.org
     cert.get_subject().OU = options.department
     cert.get_subject().CN = options.hostname
+    cert.add_extensions([
+        crypto.X509Extension(
+            b"subjectAltName",
```
Comment (smarlowucf) [inline regionService/usr/sbin/genRegionServerCert]: Single quote to stay consistent.

**Review by smarlowucf — APPROVED** [review]
Looks good

File: regionService/cloud-regionsrv.spec (hunk 1 above)
Comment (rjschwei) [reply regionService/cloud-regionsrv.spec:32]: they are now python3- dependencies and netaddr has been replaced by ipaddress which is built in

File: regionService/srv/www/regionService/regionInfo.py (hunk 2 above)
Comment (rjschwei) [reply regionService/srv/www/regionService/regionInfo.py:135]: variable names get fixed when the IPv6 support gets implemented, also hopefully this week

File: regionService/usr/sbin/genRegionServerCert (hunk 3 above)
Comment (rjschwei) [reply regionService/usr/sbin/genRegionServerCert]: fixed thanks

**Pull request merged**